sys/bloom/bloom.c

```diff
@@ -35,7 +35,7 @@ bloom_t *bloom_new(size_t size, size_t num_hashes, ...)
     }
 
     /* Allocate Bloom array */
-    if (!(bloom->a = calloc(ROUND(size), sizeof(char)))) {
+    if (!(bloom->a = calloc(sizeof(uint8_t), ROUND(size)))) {
```

Comment:
While it makes not really a difference which order you choose here, I'd say the current order is the semantically more correct one. The bloom filter buffer operates on `nmemb` different `uint8_t`s, hence `size = sizeof(uint8_t)` and `nmemb = ROUND(size)`